src/directives/gmf-map.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @fileoverview This file provides the "map" directive
+ * for GeoMapFish applications.
+ *
+ * Example:
+ *
+ * <ngeo-gmf-map ngeo-gmf-map-map="::mainCtrl.map"></ngeo-gmf-map>
+ */
+goog.provide('ngeo.gmf.mapDirective');
+
+goog.require('goog.asserts');
+goog.require('ngeo.Debounce');
+goog.require('ngeo.Location');
+goog.require('ngeo.mapDirective');
+
+
+/**
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+ngeo.gmf.mapDirective = function() {
+  return {
+    scope: {
+      'map': '=ngeoGmfMapMap'
+    },
+    bindToController: true,
```

Comment:
@fgravin Thanks for the tip, but if I do so the page fails loading with error:

``
.
angular.js:12330 AssertionError: Assertion failed
    at new goog.asserts.AssertionError (http://localhost:3000/@/home/alex/Bureau/ngeo/node_modules/openlayers/node_…2b6477e735055b8f53284d5cf6a4af1b29da/closure/goog/asserts/asserts.js:62:20)
    at Object.goog.asserts.doAssertFailure_ (http://localhost:3000/@/home/alex/Bureau/ngeo/node_modules/openlayers/node_…b6477e735055b8f53284d5cf6a4af1b29da/closure/goog/asserts/asserts.js:119:11)
    at Object.goog.asserts.assert [as assert] (http://localhost:3000/@/home/alex/Bureau/ngeo/node_modules/openlayers/node_…b6477e735055b8f53284d5cf6a4af1b29da/closure/goog/asserts/asserts.js:149:18)
    at new ngeo.gmf.MapController (http://localhost:3000/@/home/alex/Bureau/ngeo/src/directives/gmf-map.js:48:16)
    at Object.invoke (http://localhost:3000/node_modules/angular/angular.js:4450:17)
    at $get.extend.instance (http://localhost:3000/node_modules/angular/angular.js:8999:34)
    at nodeLinkFn (http://localhost:3000/node_modules/angular/angular.js:8109:36)
    at compositeLinkFn (http://localhost:3000/node_modules/angular/angular.js:7541:13)
    at nodeLinkFn (http://localhost:3000/node_modules/angular/angular.js:8145:24)
    at compositeLinkFn (http://localhost:3000/node_modules/angular/angular.js:7541:13)
``

It seems that ``this['map']`` is then not available at
https://github.com/camptocamp/ngeo/blob/gmf-map/src/directives/gmf-map.js#L48
What's the correct way to access the map property in that case?